UpdatePipelineLabel/UpdatePipelineLabel/LabelUpdateHelper.cs

```diff
@@ -0,0 +1,197 @@
+﻿using Azure;
+using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using ZenHub;
+using ZenHub.Models;
+
+
+
+namespace UpdatePipelineLabel
+{
+    class LabelUpdateHelper
+    {
+        private const string WorkspaceName = "NuGet Client Team";
+        private readonly GitHubClient GitHubClient;
+        private readonly ZenHubClient ZenHubClient;
+
+        private string repoName;
+        private string githubToken;
+        private string zenhubToken;
+        private int issueNumberFrom;
+        private int issueNumberTo;
+
+        //If specify a range in the command, then only run issues having issue numbers in this range.
+        //If range is not specified, run all the issues in the repository.
+        bool processAll;
+
+        //private SummaryReport summaryReport;
```

Comment:
leftover comment?

Reply:
Thanks for reviewing! Yes, should be removed.